frontend/app/controllers/skill-search.js

```diff
@@ -19,13 +26,42 @@ export default class SkillSearchController extends Controller {
   @computed("model")
   get selectedSkill() {
     const skillId = this.router.currentRoute.queryParams.skill_id;
+    this.currentSkillId = skillId;
     return skillId ? this.get("store").peekRecord("skill", skillId) : null;
   }
 
+  @action
+  updateSelection() {
```

Comment:
Change this to `filterPeopleSkills(skillId, level)`. Then you can call this action from both the dropdown and the slider.

Reply:
Not quite sure how this is supposed to work: the action setSkill requires a skill, not an id - this skill is only accessible from the dropdown and isn't saved anywhere else, as such the slider can't access it. We would need at least a second action to set the skill id.

Reply:
disregard; refractored to be called via observers due to issues with the slider.